test-data/unit/check-attr.test

```diff
@@ -1731,10 +1731,29 @@ class C:
     name: Union[str, None] = attr.ib(default=None)
     options: Mapping[str, Mapping[str, Any]] = attr.ib(
         default=None, converter=default_if_none(factory=dict) \
-        # E: Unsupported converter, only named functions and types are currently supported
+        # E: Unsupported converter, only named functions, types and lambdas are currently supported
     )
 [builtins fixtures/dict.pyi]
 
+[case testAttrsUnannotatedConverter]
+import attr
+
+def foo(value):
+    return value.split()
+
+@attr.s
+class Bar:
+    field = attr.ib(default=None, converter=foo)
```

Comment:
Could you add some tests to ensure the behavior is correct here?

For example, use `reveal_type(i.field)` on an instance of `Bar`. Construct an instance with `Bar(1)` or some other type.

Reply:
Good idea, thank you! Is this better now?